plugins/optimization-detective/helper.php

```diff
@@ -60,6 +60,40 @@ function od_generate_media_query( ?int $minimum_viewport_width, ?int $maximum_vi
 	}
 }
 
+/**
+ * Gets the reasons why Optimization Detective is disabled for the current response.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return array<string, string> Array of disabled reason codes and their messages.
+ */
+function od_get_disabled_reasons(): array {
+	$conditions = array(
+		'can_optimize_response_false' => array(
+			'test'   => od_can_optimize_response(),
```

Comment:
I just realized that the disabled reasons are somewhat nested here. There are 7 different possible reasons for OD being disabled in `od_can_optimize_response()`. Should the logic in that function be similarly structured in an array in another function so that its reasons can be exposed as well? Maybe this is overkill. But I just wanted to mention it since it came to mind.